src/cmd/ksh93/bltins/math.c

```diff
@@ -147,7 +151,11 @@ const struct mathtab shtab_math[] = {{"\001acos", (Math_f)acosl},
                                      {"\001erfc", (Math_f)erfcl},
                                      {"\001exp", (Math_f)expl},
                                      {"\001exp2", (Math_f)exp2l},
+#if _lib_expm1l
                                      {"\001expm1", (Math_f)expm1l},
+#else
+                                     {"\001expm1", (Math_f)expm1},
```

Comment:
This isn't legal. The code in *arith.c* will pass a single argument of type `long double` to `expm1`. The only way this would work is if `long double` is the same as `double` on NetBSD. Which the C standard allows but I doubt it is true given the NetBSD has long double variants for the other functions. You have to create a wrapper function; e.g., as was done for `j0` via `local_j0` since `j0` takes a `double` not a `long double`.

Reply:
Fixed.